test/perf/micro/perf.jl

```diff
@@ -39,11 +39,15 @@ A = ones(200,200)
 
 ## mandelbrot set: complex arithmetic and comprehensions ##
 
+function myabs2(z)
```

Comment:
Maybe call it `cabs2`as for C and lua? (Similar for javascript `complex_abs2` -> `cabs2`)

Reply:
Actually, I would call it `cabs2`in all languages is the name doesn't exists already in any of them.

Reply:
There's a fair amount of inconsistency in function and variable names throughout the perf/micro benchmarks. `quicksort` versus `qsort`, `mat_mul` versus `matmul`, etc. For the `cabs2` functions I made the most obvious  modification of the name of the existing `abs` function in the given language (e.g. `Mod` -> `Mod2` for R, `complex_abs` -> `complex_abs2` in JavaScript). 

It'd be a good thing to have all these names uniform across all languages, but I'm not sure it's worth the effort (60ish benchmark functions, their helper funcs and variables). And short of that, I don't feel doing it just for `cabs2` is a such a great idea, since it introduces name clashes within each language.

Reply:
Fair enough, let's just go ahead with this as is.